applications/nrf_desktop/src/modules/Kconfig.dfu_mcumgr

```diff
@@ -58,6 +59,7 @@ config DESKTOP_DFU_BACKEND_SUIT
 	select MCUMGR_SMP_LEGACY_RC_BEHAVIOUR
 	select MGMT_SUITFU
 	select MGMT_SUITFU_GRP_SUIT
+	select MGMT_SUITFU_GRP_OS_BOOTLOADER_INFO_HOOK
```

Comment:
The select is no longer needed (the Kconfig is enabled by default). Whole point of my change added to SUITFU was to make it work out-of-the-box with no extra configuration on application side needed.